src/modules/add-site/components/blueprints.css

```
@@ -0,0 +1,28 @@
+.blueprints-container .dataviews-title-field {
+	@apply px-5 w-full;
+}
+.blueprints-container .dataviews-view-grid__card {
+	@apply cursor-pointer relative justify-between outline outline-a8c-gray-5 rounded h-[279px] [@media(min-height:680px)]:h-[340px] min-w-32;
+}
+
+.blueprints-container .dataviews-view-grid__card:hover img {
+	@apply scale-110 transition-transform duration-150;
+}
```

This will apply the transition when hovering in and hovering out, making it look a little bit smoother.

```suggestion
.blueprints-container .dataviews-view-grid__card img {
	@apply transition-transform duration-150;
}

.blueprints-container .dataviews-view-grid__card:hover img {
	@apply scale-110;
}
```

---

Great point! I'm changing that

---

[86b6525](https://github.com/Automattic/studio/pull/1760/commits/86b6525d4c5467c6fd4372054712aa8a5e16bbc6) thanks for the suggestion!